### ISSUE TITLE
[Testing] Identify which browsers we can support

### ISSUE THREAD

**Issue by lin-d-hop** [openfoodfoundation/openfoodnetwork#3738]
## Description

After the completion of #3737 use the identified tool to understand which browsers and versions OFN can currently support.

Use the identified tool to run through our release testing process:
<link to come>

Then off the back of the testing document in the OFN Github Wiki the browsers and versions supported.

**Comment by Erioldoesdesign** [comment]
I think this would make a good potential first issue for a contributor looking to get more QA developer experience and documentation/wiki writing exp.

**Comment by RachL** [comment]
@Erioldoesdesign yes! We've added it to the welcome new dev board but we didn't add the good-first-issue label as we thought this was a label targeting more devs. But maybe this is not true. Do you know any other repo that uses this label for non devs as well? I'm more used to gitlab than github so I don't really know how new devs are looking at it.

**Comment by Erioldoesdesign** [comment]
@RachL from my OSS newbies exp, everyone uses 'good first issue' to look for tasks even if they are testing, doc writing and design! Though from a label perspective, it's good to have labels that also describe the nature of the issue for contributors so this one I would put labels like :
'Good first issue + documentation + QA testing' but always curious to hear from the dev team too on their OSS contributor perspective.
